app/com/gu/viewer/views/viewer.scala.html

```diff
@@ -44,6 +46,8 @@ <h1 class="top-toolbar__page-title">
         @if(previewEnv == "preview") {
             <div data-hidedesktopenabled="true" class="info-bar">To enable desktop preview you need to check and verify mobile first</div>
         }
+        <div data-showreaderactive="true" class="info-bar is-hidden">Copy taste presents you with a distraction free view of the article content, if you have any feedback please email <a href="mailto:digitalcms.dev@@guardian.co.uk">Digital CMS</a></div>
```

Comment:
Really close. Can you update to:

"Copy taste presents you with a distraction free view of the article content. If you have any feedback please email the Digital CMS team"